itests/qtest/pom.xml

```diff
@@ -118,6 +118,11 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
I think we should create our own module in itest, like `qtest-iceberg`

Reply:
I opened an upstream [jira](https://issues.apache.org/jira/browse/HIVE-25010) to track this work.